modular_skyrat/modules/gun_cargo/code/armament_datums/allstar.dm

```diff
@@ -60,3 +60,9 @@
 	item_type = /obj/item/weaponcrafting/gunkit/hellgun
 	lower_cost = CARGO_CRATE_VALUE * 5
 	upper_cost = CARGO_CRATE_VALUE * 8
+
+/datum/armament_entry/cargo_gun/allstar/gunkit/beam_rifle
+	item_type = /obj/item/weaponcrafting/gunkit/beam_rifle
+	lower_cost = CARGO_CRATE_VALUE * 6
```

Comment:
I don't know how I feel about just about anyone beig able to buy the parts for this, wasn't it something science had to research?